docs/lang/articles/contribution/contributor_guide.md

```diff
@@ -22,14 +22,10 @@ Always use straightforward (sometimes even brute-force) solutions: Complicated c
 
 ### Juxtapose pros and cons
 
-When it comes to making a design decision, weigh up its pros and cons. A design is *good to go,* so long as its advantages outweigh its disadvantages.
+When it comes to making a design decision, weigh up its pros and cons. A design is *good to go* so long as its advantages outweigh its disadvantages.
 
 > "Perfection (in design) is achieved not when there is nothing more to add, but rather when there is nothing more to take away." [Antoine de Saint-Exupéry](https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar)
```

Comment:
While the sentence stands on itself, I feel like its meaning it more like “keep things simple”. The context, however, is about “everything has pros and cons and perfectionism is often not optimal”. WDYT?

Reply:
Spot-on! This quote is misplaced. It ought to be placed under ### Be pragmatic & no overkills.